tests/integration-tests/cfn_stacks_factory.py

```diff
@@ -307,6 +307,21 @@ def delete_all_stacks(self):
                     )
                 )
 
+    def delete_stacks_retain_ad(self):
+        """Destroy all created stacks."""
+        logging.debug("Destroying all cfn stacks except for AD and VPC stacks")
+        for value in reversed(OrderedDict(self.__created_stacks).values()):
+            if "vpc" in value.name or "SimpleAD" in value.name or "MicrosoftAD" in value.name:
```

Comment:
Minor: in short we could do `if value.name in ["vpc", "SimpleAD", "MicrosoftAD"]`